docs/jax.lax.rst

```diff
@@ -196,3 +196,14 @@ Linear algebra operators (jax.lax.linalg)
     qr
     svd
     triangular_solve
+
+Argument wrappers
```

Comment:
I might call this "Types" or "Classes" or something like that. I don't find "wrappers" very descriptive.